### ISSUE TITLE
ESC doesn't bring the Pause Menu - can anyone confirm?

### ISSUE THREAD

**Issue by Infixo** [civfanatics/CQUI_Community-Edition#154]
When in the World View (no window open):
Vanilla game: ESC key brings up the Pause Menu.
CQUI: ESC doesn't do so.

**Comment by the-m4a** [comment]
No repro from me... using the current files in master (except for your PR completed this morning, which should not affect this).  

- Started a new game with just CQUI and Expansion2, ESC brought up the pause menu as expected
- Loaded EXP2 game with CQUI and Civitas, ESC brought up the pause menu
- Loaded base game (no expansions) with CQUI and ESC brought up the pause menu

That said... I think I saw this behavior (where ESC did not bring up the menu) at some point as well.  Any chance you had a savegame where this behavior is seen?  CQUI doesn't handle the ESC key, though perhaps it's somehow "stuck" and not passing it down the stack (though, looking at the code I'm not sure how that'd happen).

**Comment by Infixo** [comment]
No specific savefile. I also noticed that it doesn't happen regularly - hence my question.
Most of the time it seems to be working ok. Could also be some other mod interfering and that would point to the mods used in this USA savefile. I downloaded some mods that I don't use on a regular basis. I uninstalled them since the bug was fixed and as I am saying - seems to working good. But maybe this is just a coincidence, idk.
Anyway, I will be observing if this still happens.
